src/globals.c

```diff
@@ -6,6 +6,9 @@ int blocks_idx = 0;
 func_t *FUNCS;
 int funcs_idx = 0;
 
+trie_t *FUNCS_TRIE;
```

Comment:
Rename to `FUNC_TRIES` to emphasize a collection of trie data structure associated with functions.
In addition, add some comments.